tests/Command/CreateRequestsCommandTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Tests\Command;
+
+use donatj\MockWebServer\Response;
+use OpenEuropa\EPoetry\Console\Application;
+use OpenEuropa\EPoetry\Tests\Requests\AbstractCommandTest;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @internal
+ * @coversNothing
+ */
+final class CreateRequestsCommandTest extends AbstractCommandTest
+{
+    /**
+     * Data provider.
+     *
+     * @return mixed
+     */
+    public function createRequestsCases()
```

Comment:
If we only have one test case per class we can simple call this `public function dataProvider()`.

Reply:
:+1: